README.md

```diff
@@ -54,7 +57,7 @@ Text keys live in [Lokalise](https://lokalise.com/) and are exported from there
 
 This is the process for updating/adding text keys:
 
-1. Make updates to text keys in Lokalise, i.e. add new text keys or update translations. Preferrably we use the Figma/Sketch
+1. Make updates to text keys in Lokalise, i.e. add new text keys or update translations. Preferably we use the Figma/Sketch
```

Comment:
We actually don't use the Figma plugin _at all_ hehe.. 😳  But would be nice to use that though, so maybe we can leave that sentence to force it to implement it at some point? 😅